_includes/barnabus/link.html

```diff
@@ -1 +1 @@
- <a href="https://www.upwork.com/fl/johnbampton" target="_blank" rel="noopener">Barnabus</a>
\ No newline at end of file
+ <a href="https://www.upwork.com/fl/johnbampton"  >Barnabus</a>
```

Comment:
There should be no space inside the HTML `a` tag at the end of the tag.  

In some cases you added an extra space.  So you may need to remove 2 spaces from some lines